lib/internal/modules/esm/get_format.js

```diff
@@ -35,15 +35,15 @@ const protocolHandlers = {
 
 /**
  * @param {URL} parsed
- * @returns {string | null}
+ * @returns {string | undefined}
  */
 function getDataProtocolModuleFormat(parsed) {
   const { 1: mime } = RegExpPrototypeExec(
     /^([^/]+\/[^;,]+)(?:[^,]*?)(;base64)?,/,
     parsed.pathname,
   ) || [ null, null, null ];
 
-  return mimeToFormat(mime);
+  return mimeToFormat(mime) ?? undefined;
```

Comment:
I think all these `?? undefined` bits are redundant; doesn’t `??` only evaluate when the left hand side is undefined?

Reply:
When the left-hand side is nullish. I agree they're redundant though, because all of our code treat nullish values the same way (so it shouldn't matter if we're returning `null` or `undefined`)